### DISCUSSION TITLE
its posible not sanitize html, head, body tags ?

### DISCUSSION THREAD

**Discussion by lnazarit** [xdan/jodit#683] [Q&A] [answered]
its posible not remove html, head, body tags ?

**Comment by xdan** [comment, marked as answer]
By default Jodit edits an HTML *fragment*, so the `html`/`head`/`body` wrapper tags get stripped — that's expected for an inline editor. If you need to edit a whole document and keep those tags, turn on `editHTMLDocumentMode: true` — it runs in an iframe and preserves the full document structure. Without that mode there's no way to keep the structural tags.
